src/platform/Zephyr/ConfigurationManagerImpl.cpp

```diff
@@ -209,8 +209,8 @@ void ConfigurationManagerImpl::DoFactoryReset(intptr_t arg)
 CHIP_ERROR ConfigurationManagerImpl::GetPrimaryWiFiMACAddress(uint8_t * buf)
 {
 #if CHIP_DEVICE_CONFIG_ENABLE_WIFI
-    const net_if * const iface = InetUtils::GetInterface();
-    VerifyOrReturnError(iface != nullptr && iface->if_dev != nullptr, CHIP_ERROR_INTERNAL);
+    const net_if * const iface = InetUtils::GetWiFiInterface();
```

Comment:
@axelnxp fyi, see these Zephyr platform changes needed to support multiple network interfaces.

Reply:
Thank you Damian for the notification. We'll review internally the changes.
BTW, I didn't follow your recent changes, did you switch to the common wifi driver in platform/Zephyr ?

Reply:
Not yet, we had a lot of not contributed changes to WiFiManager on our fork so we first need to contribute them and only then we have to check if WiFiManager can be unified across our and your platform.

Reply:
OK understood. I started checking the changes here, and so far it seems everything can be leveraged in the common driver.